vita3k/cpu/include/cpu/functions.h

```diff
@@ -54,3 +66,7 @@ bool hit_breakpoint(CPUState &state);
 void trigger_breakpoint(CPUState &state);
 void log_code_add(CPUState &state);
 void log_mem_add(CPUState &state);
+
+CPUContextPtr init_cpu_context();
```

Comment:
not nesessary, include `<memory>` and do `std::make_unique<CPUContext>()`;

Reply:
I don't understand